services/scan/schema.sql

```diff
@@ -35,6 +35,7 @@ create table sheets (
   -- When adjudication is finished, these values are updated to now.
   front_finished_adjudication_at datetime,
   back_finished_adjudication_at datetime,
+  finished_adjudication_at datetime,
```

Comment:
I'm a little confused - why is there still a concept of adjudication if we removed the feature?

Reply:
We still have a concept of “this ballot has a problem” like an over vote that the user can decide something about (accept or reject). So we need this value to indicate when it was accepted. I'm happy to update the language if you can think of something better.

Reply:
Ah ok, I think that still sounds like adjudication to me! And [NIST](https://pages.nist.gov/ElectionGlossary/#:~:text=adjudication,ballots%20to%20reflect%20voter%20intent.) agrees:
<img width="875" alt="Screen Shot 2022-11-03 at 11 17 35 AM" src="https://user-images.githubusercontent.com/530106/199802856-eb22506b-96b3-4090-a935-e1ac8579d671.png">

Maybe I was just thrown off by the PR title